src/test.rs

```diff
@@ -130,8 +129,7 @@ fn select() {
 }
 
 #[test]
-///XXXjdm Windows' libc doesn't include fork.
```

Comment:
I think keeping the comment would be confusing: the multi-process tests actually can't work / don't make sense on platforms using the `inprocess` implementation -- regardless of whether `fork()` is available or not.